### PR TITLE
PAN-PA-5400-MPC-A.yaml

### PR DESCRIPTION
**Pull request by kalazzawi**

### CODE REVIEW

**Review by danner26 — CHANGES_REQUESTED** [review]

**Comment by danner26** [comment]
Please note, you do not need to put `mgmt_only: false`

**Review by kalazzawi — COMMENTED** [review]
part_number: PAN-PA-5400-MPC-A

File: module-types/Palo Alto/PAN-PA-5400-MPC-A.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: Palo Alto
+model: PAN-PA-5400-MPC-A
```
[the remark this answers is not in the document]
Comment (kalazzawi) [reply module-types/Palo Alto/PAN-PA-5400-MPC-A.yaml:3]: part_number: PAN-PA-5400-MPC-A

**Review by kalazzawi — COMMENTED** [review]
part_number: PAN-PA-5400-MPC-A

**Comment by kalazzawi** [comment]
Please review at your earliest convenience.

**Review by danner26 — APPROVED** [review]

**Pull request merged**